core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -169,6 +176,7 @@ public LockResource lockExclusive(StateLockOptions lockOptions)
       }
     }
     if (lockAcquired) { // Lock was acquired within grace-cycle.
+      LOG.debug("Thread-%d acquired the lock within grace-cycle.", Thread.currentThread().getId());
       activateInterruptCycle();
     } else { // Lock couldn't be acquired by grace-cycle.
       if (graceMode == StateLockOptions.GraceMode.TIMEOUT) {
```

Comment:
Could it be useful to log something here at INFO, since we were not able to get the lock?